src/docs/development/tools/android-studio.md

```diff
@@ -159,45 +161,82 @@ Also available in the "more actions" menu:
 * **Show Repaint Rainbow**: Shows rotating colors on layers when repainting.
 * **Hide Debug Mode Banner**: Hides the debug banner even when running a
   debug build.
-* **Auto horizontal scroll**: XXX - TBD
-* **Highlight nodes displayed in both trees**: XXX - TBD
+* **Highlight nodes displayed in both trees**: In the inspector, highlights
+  nodes shown in both the details and summary trees.
 * **Open Timeline View**: Analyzes activity of the application as it runs.
 * **Open Observatory**: Profiles Dart applications.
 
 ### Show performance data
 
 To view the performance data, including the widget rebuild information,
-start the app with **Debug**, and then open the Performance tool window
+start the app in **Debug** mode, and then open the Performance tool window
 using **View > Tool Windows > Flutter Performance**.
 
 ![Flutter performance window]({% asset tools/android-studio/widget-rebuild-info.png @path %})
 
 To see the stats about which widgets are being rebuilt, and how often,
 click **Show widget rebuild information** in the **Performance** pane.
-You can see how many times a widget was rebuilt in the last frame.
-For a high number of rebuilds, the exact count shows in the second column
-from the right, and a yellow spinning circle displays. 
-The column to the far right shows how many times a widget was rebuilt
-since entering the current screen. The app shown in this screenshot
-demonstrates poor performance.
-For widgets that weren't rebuilt, a solid grey circle displays.
+The exact count of the rebuilds for this frame displays in the second
+column from the right. For a high number of rebuilds, a yellow spinning
+circle displays. The column to the far right shows how many times a
+widget was rebuilt since entering the current screen.
+The app shown in this screenshot demonstrates poor performance.
+For widgets that aren't rebuilt, a solid grey circle displays.
```

Comment:
we should clarify here that the rebuilds aren't telling you performance is poor but giving a window into what work that might have had poor performance happens in the frame